bin/contact-search-backfill.py

```diff
@@ -1,11 +1,14 @@
 #!/usr/bin/env python
+import sys
```

Comment:
As a general rule, I believe every import should come after `gevent`'s `monkey.patch_all()`. This is explicitly mentioned in `gevent`'s docs: https://www.gevent.org/api/gevent.monkey.html.

So I think these files should start with:

```python
from gevent import monkey

monkey.patch_all()

import sys
...
```

Reply:
@jpmelos Right, fixed!